config.js

```diff
@@ -1,6 +1,6 @@
 exports.DATABASE_URL = process.env.DATABASE_URL ||
                        global.DATABASE_URL ||
-                      'mongodb://localhost/basic-auth-demo';
+                      'mongodb://localhost/jwt-auth-demo';
 exports.PORT = process.env.PORT || 8080;
 exports.JWT_SECRET = process.env.JWT_SECRET;
```

Comment:
That makes sense. My concern is about manually setting JWT_SECRT in package.json for test command. That's fine, just need to make sure to tell them to do that.

Alternatively, could set default only if in test environment.